InternshipProject#2/Helpers/TokenRevocation.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace InternshipProject_2.Helpers;
+
+public class TokenRevocation
```

Comment:
I don't really see how this revocation stuff working.
BTW: you don't need to create a second helper for this. Use the existing Token helper, you might need to rename it.

Reply:
Redid logout